README.md

```diff
@@ -6,7 +6,7 @@ A GStreamer element to measure framerate, bitrate and CPU usage
 
 On Debian-based systems:
 ```bash
-./autogen
+./autogen.sh
 ./configure --prefix /usr/ --libdir /usr/lib/x86_64-linux-gnu/
```

Comment:
Could you also modify the `--libdir`, please. It is sometimes annoying when installing on a jetson

Reply:
You can use:

```bash
./configure --prefix /usr/ --libdir /usr/lib/$(uname -m)-linux-gnu/
```

instead